ecs-image-build/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG IMAGE_VERSION="latest"
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-open-jdk8-maven-build:${IMAGE_VERSION}
```

Comment:
shouldn't this be one of the "runtime" base images we've been building?

Reply:
The runtime images we've recently built are jdk7 not jdk8. The application requires jdk8

Reply:
Is the Java17 runtime image incompatible with the Java8 built binary? I'm pretty sure (though happy to be proven wrong!) that we can run the binary in the new java17 runtime container even if it was built with java 8

Reply:
416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-open-jdk-runtime-17 doesn't start up the application as expected.

Reply:
Confirmed that the springboot version its using (2.2.0) is too old for java 17 runtime so even though it should be able to run when built from java 8 image we'd need to update the service itself to use a newer version of springboot which, from a quick google, looks like 2.5.5 https://docs.spring.io/spring-boot/docs/2.5.5/reference/html/getting-started.html#getting-started.system-requirements

This can go in with the jdk8 build container as a base image for now providing we have an item in the backlog to tackle this change asap